apps/block_scout_web/test/block_scout_web/channels/exchange_rate_channel_test.exs

```diff
@@ -28,7 +28,7 @@ defmodule BlockScoutWeb.ExchangeRateChannelTest do
       last_updated: DateTime.utc_now(),
       market_cap_usd: Decimal.new("1000000.0"),
       name: "test",
-      symbol: "cGLD",
```

Comment:
id just check that this is only effecting the UI and not any api that might be served. As I dont think we are ready to change cross service apis yet